config/config_test.go

```diff
@@ -345,21 +346,35 @@ func TestConsensusUpgrades(t *testing.T) {
 	currentVersionName := protocol.ConsensusV7
 	latestVersionName := protocol.ConsensusCurrentVersion
 
-	leadsTo := consensusUpgradesTo(a, currentVersionName, latestVersionName)
+	leadsTo := consensusUpgradesTo(a, currentVersionName, latestVersionName, checkConsensusVersionName)
 	a.True(leadsTo, "Consensus protocol must have upgrade path from %v to %v", currentVersionName, latestVersionName)
 }
 
-func consensusUpgradesTo(a *require.Assertions, currentName, targetName protocol.ConsensusVersion) bool {
+func checkConsensusVersionName(a *require.Assertions, name string) {
+	// ensure versions come from official specs repo
```

Comment:
why is this check important?

Reply:
I believe this is a sanity check to ensure we don't accidentally enable vAlpha in official builds.

Reply:
yes it was an idea I had before merging these alternate consensus versions, that I should add something to be extra careful to ensure a version like "alpha3" doesn't make it into the path from v7 to ConsensusCurrentVersion and into a release